### ISSUE TITLE
Test: Screen reader unable to read the items in the status bar

### ISSUE THREAD

**Issue by karthiknadig** [microsoft/vscode#177896]
Testing #177674

Encountered this on Linux Mint with Orca screen reader.

**Comment by meganrogge** [comment]
Can you provide more details? Which status bar items were you trying to read?

**Comment by karthiknadig** [comment]
Tabbing out of the Terminal into status bar:
![image](https://user-images.githubusercontent.com/3840081/226768087-f614ff5a-3c7d-421e-851f-3d830fe28885.png)

All that the screen reader says is "Status Bar", it is not able to read anything from there, or the contents of the language status.

I am also not able to tab to any specific item there. Tabbing out of the terminal to status bar selects the whole status bar as seen in screen shot.

**Comment by bpasero** [comment]
🤷 , @isidorn worked on accessibility before so maybe he remembers

**Comment by isidorn** [comment]
@karthiknadig what happens if you tab into the status bar and you use left / right to navigate into it. Does that work?

The idea behind this behaviour is that the status bar acts like all other activity bars. It is just one tab stop - to make the general tab navigation smoother.

**Comment by meganrogge** [comment]
It's working for me with left and right arrow navigation.

**Comment by Tyriar** [comment]
Guess it was confusion around tab vs arrow navigation depending on the component.

**Comment by karthiknadig** [comment]
When I tab to the terminal, it gives me instructions on what to do to make use of screen reader features. May be tabbing into the status bar should say that you can use arrow to navigate through the items in it?

**Comment by isidorn** [comment]
@karthiknadig I like that suggestion.
Code pointer for a PR https://github.com/microsoft/vscode/blob/2c01332c081ec4037a3dec3d0da73fcd3be737a9/src/vs/workbench/browser/parts/statusbar/statusbarPart.ts#L297

**Comment by isidorn** [comment]
@meganrogge none of the users complained. I think this is expected behaviour from an item that has a `statusbar` role.

**Comment by meganrogge** [comment]
Ok then IMO, we shouldn't add anything extra here
